schemas/com.snowplowanalytics.telemetry/collector_telemetry/jsonschema/1-0-0

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for collector telemetry",
+  "self": {
+    "vendor": "com.snowplowanalytics.telemetry",
+    "name": "collector_telemetry",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "collectorHost": {
+      "type": "string",
+      "description": "Host URL of the collector, e.g. https://collector.acme.com",
+      "format": "uri"
```

Comment:
If this always includes the protocol I think we're fine - just need to be cautious if it's host only to avoid invalidating against `localhost`, IP addresses etc.

Reply:
Good point, I think localhost will most likely, so I'll remove the URI format.